features/networking/sdn.feature

```diff
@@ -51,11 +51,12 @@ Feature: SDN related networking scenarios
   @admin
   Scenario: an empty OPENSHIFT-ADMIN-OUTPUT-RULES chain is created in filter table at startup
     Given the master version >= "3.6"
-    Given I select a random node's host
-    And the node service is verified
+    Given I have a project
+    Given I have a pod-for-ping in the project
+    Then evaluation of `pod.node_name(user: user)` is stored in the :node_name clipboard
```

Comment:
`(user: user)` shouldn't be needed. it is the default anyway.

Reply:
Thanks Akostadinov, remove (user: user).